src/Components/NewAdminForm/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+// import PropTypes from 'prop-types';
+import styles from './style.module.css';
+import Icon from '../Icon';
+
+function NewAdminForm() {
+  const initialFormData = {
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: '',
+  };
+  const [formData, setFormData] = useState(initialFormData);
+
+  const [showPassword, setShowPassword] = useState(false);
+
+  function handlePassword() {
+    setShowPassword(!showPassword);
+  }
+
+  function handleOnChange(e) {
+    const key = e.target.name;
+    const { value } = e.target;
+    setFormData({
+      ...formData,
+      [key]: value,
+    });
+  }
+
+  return (
+    <div className={styles.container}>
+      <h3>Add New Admin</h3>
+      <form className={styles.newAdminForm}>
+        <label htmlFor="firstName" className={styles.fieldContainer}>
+          <div className={styles.labelName}>
+            First Name
+          </div>
+          <input
+            id="firstName"
+            onChange={handleOnChange}
+            type="text"
+            name="firstName"
+            placeholder="Enter first name"
```

Comment:
Sorry, I know its not reflected on figma but i've been trying to get UX team to update. These all should just not have "Enter" included. e.g. `last name` instead of `Enter last name`